src/Authentication/Rps/IRpsConfig.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="IRpsConfig.cs" company="ProjectBlackmagic">
+// RestfulClient
+// Copyright (c) ProjectBlackmagic. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Github: https://github.com/ProjectBlackmagic/RestfulClient.
+// </copyright>
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace ProjectBlackmagic.RestfulClient.Authentication.Rps
+{
+    /// <summary>
+    /// Interface for configuration for RPS (Live ID) authentication.
+    /// </summary>
+    public interface IRpsConfig
+    {
+        /// <summary>
+        /// Gets the authentication url.
+        /// </summary>
+        /// <value>
```

Comment:
I don't think the value param is required for properties. It might be little bit verbose.